mautic/mautic
# Change campaigns with unpublished campaign in select field

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
PR added small improvements to change campaigns. 
Allow list all campaigns include unpublished. 
This change allow work editors with campaign during the test/inactive mode.

#### Steps to test this PR:
1.  Create two campaigns - first published, second unpublished
2.  Create new campaign and create action Change campaigns
3.  Works  If you see both campaign in select list (published, and unpublished)
